### PR TITLE
Mangal position setting added

### LINKED ISSUE #557: Double Mangals Appearing in SG Banis
Some Banis are showing mangals twice. Need to add an option to choose mangal position (similar to the setting that exists for SG mobile app)

Screen Shot 2019-04-01 at 2 01 01 PM

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2019-04-14 at 1 26 34 AM

Fixes #557 

* [x] Add mangal position settings 
* [x] Add the blacklist logic for showing blank and null values of mangalPosition column
* [x] Reload Bani when mangal position setting is changed
* [x] Add resume functionality to reload bani, so that bani reloads from the exact place where the setting was changed.

### CODE REVIEW

**Comment by tarunsingh5** [comment]
@inderpreetsingh what is the default mangal setting? it should be "current saroops" as default

**Comment by inderpreetsingh** [comment]
@tarunsingh5 Yes, that's what it is. 🙏

**Comment by tarunsingh5** [comment]
> @tarunsingh5 Yes, that's what it is. 🙏

awesome :)

File: www/js/defaults.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
         "display-visraams": false
       },
       "gurbani": {
-        "bani-length": "short"
+        "bani-length": "short",
+        "mangal-position": "above"
```
[the remark this answers is not in the document]
Comment (inderpreetsingh) [reply www/js/defaults.json]: Gosh! I could swear it was current :D Yes, let me fix it.

**Comment by tsingh777** [comment]
lets get https://github.com/KhalisFoundation/sttm-desktop/pull/576 in first

**Comment by maneetpaul** [comment]
This will be part of 5.1, please hold for now.

**Comment by tsingh777** [comment]
@maneetpaul i would say add this since we see double mangals without it

**Pull request merged**